.github/workflows/ci_integration.yml

```diff
@@ -278,7 +278,7 @@ jobs:
           export TAG_LATEST_FOR_BRANCH="false"
           export TAG_LATEST_FOR_PY="false"
 
-          if [[ $BRANCH == "ltm-1.6" ]]; then
+          if [[ $BRANCH == "develop" ]]; then
```

Comment:
I think this has caused some confusion with the 2.0 update for apps, can we perform an audit of apps ltm-1.6 and develop branch to ensure this will not cause any issues?

Reply:
> apps upstream CI should be explicitly using 

The operative word being `should`, so if I understand correctly, if the app owners did not complete, know, or are  not perfect in their application of said policy, it sounds like you are saying this would cause issues again.

Reply:
The "App Upstream Testing" CI workflow uses this base provided in Core: https://github.com/nautobot/nautobot/blob/develop/.github/workflows/plugin_upstream_testing_base.yml which explicitly tests against `develop` and `ltm-1.6` (which is the tag from `ltm/1.6` and that branch needs to be updated, see #4638).

As far as what App's have in their CIs and dev environment: they should be using more fitting names to test against. `latest` having always tracked potentially buggy, unreleased code (hence upstream monitoring) could hold up apps unnecessarily. 

It would appear a lot of this was fixed but I'm sure that's in-part to drift manager, looking at: https://github.com/nautobot/nautobot-plugin-golden-config/blob/develop/.github/workflows/ci.yml#L122-L137

Paging @whitej6 @cmsirbu @snaselj this probably something we should clean up for App's LTM-tracked and 2.0-tracked CI workflows:

- `stable` and `2.0.0` for 2.0-track
- `ltm-1.6` and `1.6.0` for LTM-track